### ISSUE TITLE
Forbid to use `+=` with list arguments

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#870]
# Rule request

## Thesis

We can detect when people write code like this: 

```python
some_list += [1, 2, 3]
```

What happens here? `+=` works like `append()` method.
But, why do you want to use this `+=` stuff at all? Why not use just `.append()`?
It is inconsistent and should be forbidden.

**Comment by sathachao** [comment]
Hi! Can I try this one?

**Comment by sobolevn** [comment]
Sure, @sathachao! Please feel free to ask for help in case you need it.

**Comment by sobolevn** [comment]
I had to remove this violation. Since it will be included into `typed-linter` project.
